packages/trackerless-network/src/logic/protocol-integration/stream-message/StreamMessageTranslator.ts

```diff
@@ -7,30 +7,56 @@ import {
     GroupKeyRequest as OldGroupKeyRequest,
     GroupKeyResponse as OldGroupKeyResponse,
     StreamID,
+    EncryptionType as OldEncryptionType,
+    ContentType as OldContentType
 } from '@streamr/protocol'
 import {
-    ContentMessage,
+    ContentType,
     EncryptedGroupKey,
+    EncryptionType,
     GroupKeyRequest,
     GroupKeyResponse,
     MessageRef,
     StreamMessage,
     StreamMessageType
 } from '../../../proto/packages/trackerless-network/protos/NetworkRpc'
-import { ContentMessageTranslator } from './ContentMessageTranslator'
 import { EthereumAddress } from '@streamr/utils'
 import { GroupKeyRequestTranslator } from './GroupKeyRequestTranslator'
 import { GroupKeyResponseTranslator } from './GroupKeyResponseTranslator'
+import { toBinary, toUTF8 } from '../../utils'
+
+const oldToNewEnryptionType = (type: OldEncryptionType): EncryptionType => {
```

Comment:
When we do the follow-up PR to this of adding binary payload support to `streamr-client` we should think about removing the the `RSA` from the `OldEncryptionType` there as well.